src/test/kotlin/org/elm/ide/inspections/ElmIncompletePatternInspectionTest.kt

```diff
@@ -90,6 +90,137 @@ foo it =
             --EOL
 """)
 
+//    fun `test no existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (<error>case{-caret-}</error> it of<error>)</error>
```

Comment:
The last `error` here was required for the test text to be parsed. I assume it's part of the same parsing problem I described in the PR text.